### PR TITLE
Add /get_wallet and /get_sila_balance docs

### PR DESCRIPTION
**Pull request by freemish**

### CODE REVIEW

File: source/includes/_endpoints.md [hunk 1]
```diff
@@ -1271,6 +1271,89 @@ The `success` attribute is a JSON key sent in the response body.
...
+- nickname
+- whether it is a default destination
+- whether it is whitelisted
+- current token balance
+
+The wallet/address to be examined is the one derived from the `usersignature` header.
+
+### Requests
+
+Both `authsignature` and `usersignature` headers are required for this request. (The `usersignature` header should be generated with the keypair associated with the user; see [authentication](#authentication) section for more details.)
```
Comment (johndburk) [inline source/includes/_endpoints.md]: Should this be more explicit that the particular wallet bc_address is the one that should be used to sign the request (have I got that right?)?

File: source/includes/_endpoints.md [hunk 2]
```diff
@@ -1584,6 +1667,64 @@ Only an `authsignature` header is required for this request.
...
+// Java example coming soon
+```
+
+```go
+// Go example coming soon
+```
+
+*Checks token balance on a blockchain address.*
+
+This endpoint allows deletion of a user's wallet. This action cannot be performed on a user's only wallet, a user's default destination wallet, or an a wallet with a non-zero balance.
```
Comment (johndburk) [inline source/includes/_endpoints.md]: Is this a copy&paste error?
> This endpoint allows deletion of a user's wallet. This action cannot be performed on a user's only wallet, a user's default destination wallet, or an a wallet with a non-zero balance.

File: source/includes/_endpoints.md (hunk 1 above)
Comment (freemish) [reply source/includes/_endpoints.md]: yes, that's right. above that, there's a line that reads `The wallet/address to be examined is the one derived from the `usersignature` header.`. how would you recommend I make that more clear/visible?

File: source/includes/_endpoints.md (hunk 2 above)
Comment (freemish) [reply source/includes/_endpoints.md]: lol, yes it was! thanks for the spot. have addressed this

**Comment by freemish** [comment]
These endpoints aren't deployed yet (so _do not merge_), but note to self to clarify wording on /get_wallet.

**Comment by freemish** [comment]
Thanks for the re-review! Now, for my personal sanity, I'll close this PR and re-open/merge it when it comes time to release these new endpoints.

**Comment by freemish** [comment]
Re-opening since release is imminent.

**Pull request merged**